wherebyus/newspack-newsletters
# Issue 5047

### All Submissions:






### Changes proposed in this Pull Request:



Closes # .

### How to test the changes in this Pull Request:

1. Add a valid letterhead api key (I have a testing one)
2. Make a newsletter for, say, January 11, 2021
3. Write some stuff and send yourself a test email. You should expect to see at the top of the email you receive an ad.

### Other information:
